rmb-sdk-go/peer/peer.go

```diff
@@ -462,19 +483,25 @@ func (d *Peer) makeEnvelope(id string, dest uint32, session *string, cmd *string
 }
 
 func (d *Peer) send(ctx context.Context, request *types.Envelope) error {
-
 	bytes, err := proto.Marshal(request)
 	if err != nil {
 		return err
 	}
 
-	select {
-	case d.writer <- bytes:
-	case <-ctx.Done():
-		return ctx.Err()
+	var errs error
+	for _, con := range d.cons {
```

Comment:
may be should be a ring instead of a list, so on next send u try to use another relay? to balance message sending over multiple relays! Not sure if this is the best approach

Reply:
Result of connections used after using ring: 

```
con.url: wss://relay.dev.grid.tf
con.url: wss://relay.02.dev.grid.tf
con.url: wss://relay.dev.grid.tf
con.url: wss://relay.02.dev.grid.tf
con.url: wss://relay.dev.grid.tf
con.url: wss://relay.02.dev.grid.tf
con.url: wss://relay.dev.grid.tf
con.url: wss://relay.02.dev.grid.tf
con.url: wss://relay.dev.grid.tf
con.url: wss://relay.02.dev.grid.tf
```